### ISSUE TITLE
Crash In Combination With Pufferfish's Skills Mod

### ISSUE THREAD

**Issue by Moozipan** [Xires87/MobVariants#2]
I'm getting a guaranteed crash to desktop when attacking either the Explorer, or Armored Spider mob added by this mod, regardless of weapons or fists used to attack. All other mobs are fine. And it only happens in combination with the Pufferfish's Skills mod. The crash report says it got something to do with knockback, but it also happens with weapons that don't deal knockback damage.

I'm on Fabric 1.20.1, using v4.3.2 of Mob Variants and v0.10.0 of Pufferfish's Skills (https://www.curseforge.com/minecraft/mc-mods/puffish-skills).

Crash report:
---- Minecraft Crash Report ----
// Surprise! Haha. Well, this is awkward.

Time: 2023-11-22 02:02:10
Description: Unexpected error

java.lang.IllegalArgumentException: Can't find attribute minecraft:generic.attack_knockback
	at net.minecraft.class_5132.method_26865(class_5132.java:22)
	at net.minecraft.class_5132.method_26862(class_5132.java:28)
	at net.minecraft.class_5131.method_26852(class_5131.java:76)
	at net.minecraft.class_1309.method_26825(class_1309.java:1904)
	at net.minecraft.class_1657.wrapVariable$hok000$porting_lib_entity$triggerKnockbackCode(class_1657.java:28276)
	at net.minecraft.class_1657.method_7324(class_1657.java:1265)
	at net.minecraft.class_636.method_2918(class_636.java:412)
	at net.minecraft.class_310.method_1536(class_310.java:1729)
	at net.minecraft.class_310.method_1508(class_310.java:2049)
	at net.minecraft.class_310.method_1574(class_310.java:1883)
	at net.minecraft.class_310.method_1523(class_310.java:1181)
	at net.minecraft.class_310.method_1514(class_310.java:802)
	at net.minecraft.client.main.Main.main(Main.java:250)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.quiltmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:551)
	at org.quiltmc.loader.impl.launch.knot.Knot.launch(Knot.java:84)
	at org.quiltmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:28)

**Comment by Xires87** [comment]
It must be something else. Everything works correctly for me.
Please, make sure to uninstall all unnecessary mods while looking for the one that is incompatible.

Let me know if it still happens with only Mob Variants, Pufferfish's Skills, Cloth Config and Fabric API installed, and if it doesn't, find  that other incompatible mod (or mods).

**Comment by Moozipan** [comment]
It's a strange bug and I can't even reproduce it with the Pufferfish's Skills mod on my end anymore, even though I already did what you suggested last night to confirm the cause. I have over 200 mods installed and after trying endless combinations today, I identified that there are more than 3 mods involved that cause the crash, and it's fixed if I remove any of them. So I can only assume it's a very weird chain reaction caused by my very specific setup, but this won't really help you with fixing anything.
It's just curious that it only happens to these two mobs, but not a big deal when I can either prevent them from spawning or pass on one of the other mods involved. Thanks for you time!
